enhancements/oc/must-gather.md

```diff
@@ -64,16 +64,24 @@ ship a second one.
 
 `must-gather` for openshift is a combination of three tools:
 
-1. A client-side `inspect` command that works like a super-get.  It has semantic understand of some resources and traverses
-   links to get interesting information beyond the current.  Pods in namespaces and logs for those pods as a for instance.
-   Currently this is `openshift-must-gather inspect`, but we are porting this to `oc adm` as experiemental in 4.3.  We may
+1. A client-side `inspect` command that works like a super-get. It has semantic understanding of some resources and traverses
+   links to get interesting information beyond the current. Pods in namespaces and logs for those pods, for instance.
+   Currently, this is `openshift-must-gather inspect`, but we are porting this to `oc adm` as experimental in 4.3. We may
    change and extend arguments over time, but the intent of the command will remain.
-2. The openshift-must-gather image, produced from https://github.com/openshift/must-gather.  The entry point is a
+2. The openshift-must-gather image, produced from https://github.com/openshift/must-gather. The entry point is a
    [/gather bash script](https://github.com/openshift/must-gather/blob/master/collection-scripts/gather) owned by CEE
-   (not the developers) that describes what to gather.  It is tightly coupled to the openshift payload
-   and only contains logic to gather information from that payload.  We have e2e tests that make sure this functions.
+   (not the developers) that describes what to gather. It is tightly coupled to the OpenShift payload
+   and only contains logic to gather information from that payload. We have e2e tests that make sure this functions.
 3. `oc adm must-gather --image` which is a client-side tool that runs any must-gather compatible image by creating a pod,
-   running the `/usr/bin/gather` binary, and then rsyncing the `/must-gather` and includes the logs of the pod.
+   running the `/usr/bin/gather` binary, and then rsyncing the `/must-gather` and includes the logs of the pod. To 
+   avoid overwhelming the cluster by passing numerous `--image` flags or an `--all-images` flag (covered in next point),
+   we are capping this to spin up four concurrent pods. If required, it could be exposed via a flag in future.
+4. `oc adm must-gather --all-images` allows the user to perform the same task as `oc adm must-gather --image`, but 
+   instead of specifying multiple must-gather compatible images using multiple `--image` flags, it finds such images 
+   from the cluster and runs `/usr/bin/gather`. The way it finds such images is by looking up the Operators 
+   (ClusterServiceVersions/CSVs) with the annotation `operators.openshift.io/must-gather-image` on the cluster where 
```

Comment:
As mentioned in the other comment in response to question ([here](https://github.com/openshift/enhancements/pull/1487#pullrequestreview-1710161776)) `ClusterOperators` (those deployed by CVO) should also be included in this.